cmd/main.go

```diff
@@ -60,6 +61,7 @@ func main() {
 			&kustomizecommonv1.Plugin{},
 			&kustomizecommonv2.Plugin{},
 			&declarativev1.Plugin{},
+			&deployimagev1beta1.Plugin{},
 		),
```

Comment:
We need to create a new testdata sample to check the plugin. (it is here: https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/testdata/generate.sh)
The commands to run the sample are:

```
kubebuilder init 
$kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false --plugins=go/v3,deploy-config/v1-alpha1
```
Then, we need a new dir for that which will be: `project-v3-with-deploy-image`